.github/workflows/python-package.yml

```diff
@@ -48,7 +48,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: [3.6, 3.7, 3.8, 3.9, '3.10']
```

Comment:
Yaml is confusing enough as it is, but I think the quotes aren't necessary here. If they are for some reason, I think quoting every version should work and would remove a potential source of confusion.

Reply:
The quotes are unfortunately necessary. Otherwise this resolves to python v3.1 instead of 3.10. I'm fine with quotes for every version.

Reply:
Done- I put the version #s in quotes for clarity.